src/test/java/com/microsoft/sqlserver/jdbc/fedauth/PooledConnectionTest.java

```diff
@@ -51,23 +53,22 @@ public static void setupTests() throws Exception {
 
     @Test
     public void testPooledConnectionAccessTokenExpiredThenReconnectADPassword() throws SQLException {
-        // suspend 5 mins
-        testPooledConnectionAccessTokenExpiredThenReconnect((long) 5 * 60, SqlAuthentication.ActiveDirectoryPassword);
+        // suspend 60 secs
+        testPooledConnectionAccessTokenExpiredThenReconnect((long) 60, SqlAuthentication.ActiveDirectoryPassword);

```

Comment:
Are the `long` casts not redundant? See also lines 63 & 71.
```suggestion
        testPooledConnectionAccessTokenExpiredThenReconnect((60, SqlAuthentication.ActiveDirectoryPassword);
```